assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -108,6 +110,8 @@
 		--color-surface: #{$muriel-white};
 		--color-surface-backdrop: #{$muriel-gray-0};
 
+		--color-button-primary-background-hover: #{ $muriel-blue-400 };
+
```

Comment:
We'd probably want to assign a value for `--color-button-primary-scary-background-hover:` here as well, even if it's the same as the default. All CSS variables should always be present in all color schemes.

Reply:
done in f9fe03b